config/policies.js

```diff
@@ -19,24 +19,39 @@
 
 module.exports.policies = {
```

Comment:
Would it be better to do '*': ['passport', 'sessionAuth', 'isMod'] 

And then we can explicitly say which ones non-mods can view? Means that it's much harder to accidentally open something up to mods.